test-case/check-capture.sh

```diff
@@ -75,6 +75,10 @@ do
         type=$(func_pipeline_parse_value $idx type)
         snd=$(func_pipeline_parse_value $idx snd)
 
+        # Filtering feature is not enabled for pipelines dumped from proc
+        # This workaround is need for this case to run on legacy HDA platform.
```

Comment:
We cannot remove the filter here, because an empty filter means all pipelines, and filter "-f type:any" means all pipelines except WOV and Echo Reference, we cannot run arecord on these pipelines directly. I will change the filter to "-f type:any",  remove the puzzling comments, this also make this a FEATURE.